src/app/components/client/ExposureCard.tsx

```diff
@@ -70,23 +50,254 @@ type BreachExposureCategoryProps = {
+            // TODO: Add unit test when changing this code:
+            /* c8 ignore next */
+            onClick={() => setExposureCardExpanded(!exposureCardExpanded)}
+          >
+            <ChevronDown
+              // TODO: Add unit test when changing this code:
+              /* c8 ignore next */
+              className={exposureCardExpanded ? styles.isOpen : ""}
+              alt={
+                // TODO: Add unit test when changing this code:
+                /* c8 ignore next 2 */
+                exposureCardExpanded
+                  ? l10n.getString("chevron-up-alt")
+                  : l10n.getString("chevron-down-alt")
+              }
+              width="20"
+              height="20"
+            />
+          </button>
+        </div>
+        <div
+          className={`${styles.exposureDetailsSection} ${
+            // TODO: Add unit test when changing this code:
+            /* c8 ignore next */
+            exposureCardExpanded ? styles.isOpen : ""
+          }`}
+        >
+          <div>
+            <p>
+              {l10n.getFragment(
+                "exposure-card-description-info-for-sale-part-one",
+                {
+                  elems: {
+                    data_broker_link: <a href={scanResult.link} />,
+                  },
+                }
+              )}
+              <a href={scanResult.link}>
+                <span>
+                  <OpenInNew
+                    alt={l10n.getString("open-in-new-tab-alt")}
+                    width="13"
+                    height="13"
+                  />
+                </span>
+              </a>
+              {l10n.getString(
+                "exposure-card-description-info-for-sale-part-two"
+              )}
+            </p>
+          </div>
+          <div className={styles.exposedInfoContainer}>
+            <div className={styles.exposedInfoWrapper}>
+              <p className={styles.exposedInfoTitle}>
+                {l10n.getString("exposure-card-your-exposed-info")}
+              </p>
+              <dl className={styles.dataClassesList}>
+                {exposureCategoriesArray.map((item) => (
+                  <React.Fragment key={item.key}>{item}</React.Fragment>
+                ))}
+              </dl>
+            </div>
+            {isPremiumBrokerRemovalEnabled && props.scanResult.status === "new"
+              ? letsFixItBtn
+              : null}
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+
+  return exposureCard;
 };
 
-export const ExposureCard = (props: ExposureCardProps) => {
-  const {
-    exposureImg,
-    exposureName,
-    exposureData,
-    exposureDetailsLink,
-    statusPillType,
-    locale,
-    color,
-    featureFlagsEnabled,
-  } = props;
+export type SubscriberBreachCardProps = {
+  exposureImg?: StaticImageData;
+  subscriberBreach: SubscriberBreach;
+  locale: string;
+};
+const SubscriberBreachCard = (props: SubscriberBreachCardProps) => {
```

Comment:
Most of this component is also a verbatim copy of the old `<ExposureCard>` component, but only the branches relating to subscriber breaches.